docs/news/2026-07-07-cueweb-full-cuegui-parity-release.md

```
@@ -0,0 +1,222 @@
+---
+layout: default
+title: "July 7, 2026: Announcing CueWeb: The Complete Web-Based OpenCue GUI"
+parent: News
+nav_order: 0
+---
+
+# Announcing CueWeb: The Complete Web-Based OpenCue GUI
+
+### First Full Release: CueWeb Now Replicates All of CueGUI (Cuetopia and CueCommander)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)